src/main/java/net/snowflake/client/config/SFClientConfig.java

```diff
@@ -1,11 +1,21 @@
 package net.snowflake.client.config;
 
+import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.LinkedHashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
 import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import net.snowflake.client.core.SnowflakeJdbcInternalApi;
 
 /** POJO class for Snowflake's client config. */
 public class SFClientConfig {
+  // Used to keep the unknown properties when deserializing
```

Comment:
changes in this file looks the same as in https://github.com/snowflakedb/snowflake-jdbc/pull/1674 - should they repeat there?

Reply:
Sorry, fixed it.